includes/polling/mempools/edgecos.inc.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * LibreNMS 
+ *
+ * Copyright (c) 2017 Martin Zatloukal <slezi2@pvfree.net>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+if (starts_with($device['sysObjectID'], 'enterprises.259.10.1.24.')) { //ECS4510
```

Comment:
Ok so you've answered my question above, you can use the sysObjectID - you need to port this to discovery.

Can you not used the named OIDs in all of this?

You should convert these to `snmp_get_multi_oid()` when running more than one get.